### PR TITLE
statistics: remove useless code

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by winoros — APPROVED** [review]

File: pkg/statistics/histogram.go [hunk 1]
```diff
@@ -252,11 +252,6 @@ const (
 // AnalyzeFlag is set when the statistics comes from analyze.
 const AnalyzeFlag = 1
```
[the remark this answers is not in the document]
Comment (hawkingrei) [reply pkg/statistics/histogram.go:253]: But it has been as flag into the system table. system table is hard to change.

[the remark this answers is not in the document]
Comment (hawkingrei) [reply pkg/statistics/histogram.go:262]: I have reuse this variable to replace the magic code.

**Review by hi-rustin — APPROVED** [review]
Looks good to me. Thanks! 👍

**Pull request merged**